lightning/src/lib.rs

```diff
@@ -78,6 +78,7 @@ extern crate core;
 pub mod util;
 pub mod chain;
 pub mod ln;
+pub mod offers;
```

Comment:
Should we keep this private for now?

Reply:
Hmm.. I put an `allow(unused)` over the OM module and set the doc tests as `ignore` while it was private, in case that helps

Reply:
Nice. That did the trick.